content/docs/ref/pr.md

```diff
@@ -45,14 +46,14 @@ Any [generic option](/doc/ref) in addition to:
 ### Commit all files in current working directory
 
 ```cli
-$ cml pr .
+$ cml pr create .
 ```
 
 ### Automatically merge pull requests
 
 ```cli
 $ date > output.txt
-$ cml pr --auto-merge output.txt
+$ cml pr create --auto-merge output.txt
```

Comment:
I thought `--auto-merge` was deprecated in favor of `--merge`, `--squash` and `--rebase`

Reply:
```suggestion
$ cml pr create --merge output.txt  # or --squash/--rebase
```